src/commands/force/mdapi/beta/deploy.ts

```diff
@@ -0,0 +1,221 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { flags, FlagsConfig } from '@salesforce/command';
+import { Duration, once, env } from '@salesforce/kit';
+import { Messages } from '@salesforce/core';
+import { AsyncResult, DeployResult, RequestStatus, MetadataApiDeploy } from '@salesforce/source-deploy-retrieve';
+import { isString } from '@salesforce/ts-types';
+import { DeployCommand, getVersionMessage } from '../../../../deployCommand';
+import {
+  DeployAsyncResultFormatter,
+  DeployCommandAsyncResult,
+} from '../../../../formatters/deployAsyncResultFormatter';
+import { MdDeployResultFormatter, MdDeployResult } from '../../../../formatters/mdDeployResultFormatter';
+import { ProgressFormatter } from '../../../../formatters/progressFormatter';
+import { DeployProgressBarFormatter } from '../../../../formatters/deployProgressBarFormatter';
+import { DeployProgressStatusFormatter } from '../../../../formatters/deployProgressStatusFormatter';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-source', 'md.deploy');
+
+type TestLevel = 'NoTestRun' | 'RunSpecifiedTests' | 'RunLocalTests' | 'RunAllTestsInOrg';
+
+export class Deploy extends DeployCommand {
+  public static readonly description = messages.getMessage('description');
+  // TODO: change help into examples?
+  // public static readonly examples = messages.getMessage('examples').split(os.EOL);
```

Comment:
shouldn't this be uncommented?

Reply:
there weren't any in this command, even in toolbelt.  You can see their help text in `mdDeployCommandCliHelp`.

We could make examples from that, maybe?

Reply:
I discussed this with Juliet.  I'm gonna take a stab at examples and she can fix them.